### PR TITLE
Add NodeAffinity

### PR DESCRIPTION
**Pull request by dschunack**
- Add NodeAffinity to run it only on Linux Nodes with AMD64 Arch.
- Update CHANGELOG.md

Update Changelog

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Comment by dschunack** [comment]
Hi,

We have a mixed k8s with Linux and Windows Worker Nodes and from time to time the exporter tries to start on a Windows Node. It make sense to add a default node affinity to run the export only on Linux Nodes with AMD64 arch.
This will also solve the following issue: #71

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.3.1] - 2022-03-15
+
+### Added
+
+- Add NodeAffinity to run the exporter only on Linux Nodes with AMD64.
+
```
Comment (stone-z) [inline CHANGELOG.md:15]: The release heading will be added automatically
```suggestion
### Added

- Add NodeAffinity to run the exporter only on Linux Nodes with AMD64.

```

**Review by stone-z — COMMENTED** [review]
Hey @dschunack thanks a lot for taking this! One super minor request with the changelog, but I'll test this and get it in likely today.

File: CHANGELOG.md (hunk 1 above)
Comment (dschunack) [reply CHANGELOG.md:15]: I tested it on EKS today. Thanks for the fast feedback

**Pull request merged**